content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -168,16 +177,34 @@ topologyKey: kubernetes.io/hostname
 whenUnsatisfiable: ScheduleAnyway
 ```
 
+{{% notice note %}}
+topologyKey refers to the key of KubeVirt infra node labels.
+{{% /notice %}}
+
 this allows us to spread Virtual Machine equally across a cluster.
 
+**Advanced settings: customize Topology Spread Constraints and Node Affinity**
+
 However, it is possible to change the default behaviour and create your own topology combined with Node Affinity Presets.
 You can do it by expanding *ADVANCED SCHEDULING SETTINGS* on the initial nodes dashboard page.
 
 ![Instance Types and Preferences](/img/kubermatic/main/architecture/supported-providers/kubevirt/scheduling-form.png?classes=shadow,border "Advanced Scheduling Settings")
 
+{{% notice note %}}
+Node Affinity Preset Key: refers to the key of KubeVirt infra node labels.
+
+Node Affinity Preset Values: refers to the values of KubeVirt infra node labels.
+{{% /notice %}}
+
+Node Affinity Preset type is: "hard" or "soft" and refers to the same notion of [Pod affinity/anti-affinity types](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#types-of-inter-pod-affinity-and-anti-affinity):
```

Comment:
We should get rid of colons, I think once you put you should construct a list, it could be changed to
> Node Affinity Preset type can have `hard` or `soft` value...

Reply:
done